src/test/java/com/g4s8/ghman/data/DatabaseExtension.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.data;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.util.Arrays;
+import javax.sql.DataSource;
+import org.flywaydb.core.Flyway;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.postgresql.jdbc2.optional.SimpleDataSource;
+import ru.yandex.qatools.embed.postgresql.EmbeddedPostgres;
+import ru.yandex.qatools.embed.postgresql.distribution.Version;
+
+/**
+ * Database abstract case.
+ * @since 1.0
+ */
+final class DatabaseExtension implements BeforeAllCallback, AfterAllCallback {
+
+    /**
+     * Data source.
+     */
+    private static DataSource src;
```

Comment:
@olenagerasimova sorry I missed this, but please remove this static field too as well as the static method below. We don't want any static at all :)

Reply:
@victornoel I'm not sure it's possible, how we will access prepared `DataSource` from integration tests then? Please, remember that is class is used as junit [extension ](https://junit.org/junit5/docs/current/user-guide/#extensions)

Reply:
@olenagerasimova from what I read [here](https://junit.org/junit5/docs/current/user-guide/#extensions-registration-declarative), you can inject some state related to the extension into the test being executed.

Reply:
@victornoel added todo